src/components/SendAsset/SendAsset.js

```diff
@@ -168,28 +165,9 @@ const SendAsset = ({
     return updateTxFeeDebounced.cancel;
   }, [updateTxFeeDebounced]);
 
-  const handleAmountChange = (value: ?Object) => {
-    if (amount !== value?.input) setAmount(value?.input || '0');
-    if (value && assetData !== value.selector) setAssetData(value.selector);
-  };
-
   // initial
   useEffect(() => {
     resetEstimateTransaction();
-
-    if (!defaultAssetData) return;
-
-    let formattedSelectedAsset;
-    if (assetData.tokenType === COLLECTIBLES) {
-      formattedSelectedAsset = collectibles.find(({ tokenId }) => assetData.id === tokenId);
-    } else {
-      // $FlowFixMe: collectible
-      formattedSelectedAsset = assetsWithBalance.find((asset) => assetData.token === asset.token);
-    }
-
-    if (!formattedSelectedAsset) return;
-
-    handleAmountChange({ selector: formattedSelectedAsset, input: '' });
```

Comment:
All of this code seems to actually do nothing because it's run once during component mount.